hack/tools/go.mod

```diff
@@ -53,7 +53,7 @@ require (
 	github.com/fatih/color v1.14.1 // indirect
 	github.com/fatih/structtag v1.2.0 // indirect
 	github.com/firefart/nonamedreturns v1.0.4 // indirect
-	github.com/fsnotify/fsnotify v1.5.4 // indirect
+	github.com/fsnotify/fsnotify v1.6.0 // indirect
```

Comment:
The indirect dependency should not be updated manually.

And before submitting a dependency-related update, you should run the below to check if it is a valid change.
```
hack/verify-vendor.sh
hack/lint-dependencies.sh
```

To  upgrade dependency, you have to do something like 
https://github.com/pacoxu/kubernetes/blob/6385eea95be82d00dc7400cdb5fca82f1c9e76b8/hack/lint-dependencies.sh#L61-L63

for `fsnotify/fsnotify` v1.6.0 upgrade, works are done in https://github.com/kubernetes/kubernetes/pull/113037.

Reply:
With this branch, `hack/verify-vendor.sh` failed with some errors like the below:
```
=== tidying go.mod/go.sum in hack/tools
verifying github.com/fsnotify/fsnotify@v1.6.0/go.mod: checksum mismatch
	downloaded: h1:sl3t1tCWJFWoRz9R8WJCbQihKKwmorjAbSClcnxKAGw=
	go.sum:     h1:OVB6XrOHzAwXMpEM7uPOzcehqUV2UqJxmVXmkdnm1bU=

SECURITY ERROR
This download does NOT match an earlier download recorded in go.sum.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.
```